scripts/tts_engines/google_tts.py

```diff
@@ -0,0 +1,25 @@
+from google.cloud import texttospeech
+
+def narrate(input_script, output_dir):
+    output_path = output_dir + "/narration.mp3"
+
+    client = texttospeech.TextToSpeechClient()
+
+    synthesis_input = texttospeech.SynthesisInput(text=input_script)
+
+    voice = texttospeech.VoiceSelectionParams(
+        language_code="en-UK", ssml_gender=texttospeech.SsmlVoiceGender.NEUTRAL
```

Comment:
google settings to extract

Reply:
Added a note around this here: [35497ac](https://github.com/the-innovation-squad/video-generator/pull/2/commits/35497ac86c7086f4e14595ef652ef7714224aecb)